### ISSUE TITLE
[DataCap Application] <Greatdata> - <Community Earth System Model v2 ARISE>

### ISSUE THREAD

**Issue by vance8sl** [1475Notary/1475-Allocator#19]
### Data Owner Name

National Center For Atmospheric Research

### Data Owner Country/Region

United States

### Data Owner Industry

Environment

### Website

https://ncar.ucar.edu/

### Social Media Handle

Vance

### Social Media Type

Slack

### What is your role related to the dataset

Data Preparer

### Total amount of DataCap being requested

5PiB

### Expected size of single dataset (one copy)

625TiB

### Number of replicas to store

8

### Weekly allocation of DataCap requested

256TiB

### On-chain address for first allocation

f1oojpirt4mni4npobf5dcdkck2ujk3fbkbnco7di

### Data Type of Application

Public, Open Dataset (Research/Non-Profit)

### Custom multisig

- [ ] Use Custom Multisig

### Share a brief history of your project and organization

```text
I'm a member of greatdata. We have 30 members in total and 5 servers. We mainly provide service about storing data, enterprise and individual. NCAR is a center of research excellence in Earth system science sponsored by the National Science Foundation. NCAR was established by the National Science Foundation in 1960 to provide the university community with world-class facilities and services that were beyond the reach of any individual institution.
```

### Is this project associated with other projects/ecosystem stakeholders?

No

### Describe the data being stored onto Filecoin

```text
Data from ARISE-SAI Experiments with CESM2.
```

### Where was the data currently stored in this dataset sourced from

AWS Cloud

### If you are a data preparer. What is your location (Country/Region)

Singapore

### If you are a data preparer, how will the data be prepared? Please include tooling used and technical details?

```text
Tooling: Lotus, servers, harddisks.
Details: We use lotus to convert data into car files. Then we transfer these files to sps.
```

### Please share a sample of the data

```text
s3://ncar-cesm2-arise/
```

### Confirm that this is a public dataset that can be retrieved by anyone on the Network

- [X] I confirm

### What is the expected retrieval frequency for this data

Sporadic

### For how long do you plan to keep this dataset stored on Filecoin

1.5 to 2 years

### In which geographies do you plan on making storage deals

Greater China, Asia other than Greater China, North America

### How will you be distributing your data to storage providers

Cloud storage (i.e. S3), HTTP or FTP server, Shipping hard drives, Lotus built-in data transfer

### How did you find your storage providers

Slack, Big Data Exchange

### Please list the provider IDs and location of the storage providers you will be working with.

```text
f02196792 HongKong
f02247136 Beijing
f01879772 HongKong
f0103710 Guiyang
f01955030 Zhejiang
and more sps in the future
```

### How do you plan to make deals to your storage providers

Lotus client

### Can you confirm that you will follow the Fil+ guideline

Yes

**Comment by 1475Notary** [comment]
@vance8sl I can not find your application in the dashboard. You should renew or reapply your application. Please try it again.

**Comment by vance8sl** [comment]
@1475Notary I have already edited the application. Is it ok now?

**Comment by 1475Notary** [comment]
@vance8sl good job

I want to ask you some questions.

Is this your first time to apply? Have you ever applied for datacap from another allocator?
Do SPs you work with support retrieval?
Do you have enough token for your plan?
For bookkeeping, I'll keep a record of our communications. Do you accept that?

Also, please send your email to slack: 1475Notary, we need to check your identity.

**Comment by vance8sl** [comment]
@1475Notary 
Yes. But I have helped my partner to solve some problems about storing in Fil+. I have some experience.
I have learned from my partner that sp should support spark retrieval. Sps I choose will support it.
Yes. Sps have enough resource.
Yes, I accept it.

![image](https://github.com/user-attachments/assets/173f1aeb-35e6-4e8a-8fc2-cb4700b15cf6)
You can check my email to confirm my identity. Thank you.

**Comment by vance8sl** [comment]
@1475Notary I have recieved your email and I replied it.
![image](https://github.com/user-attachments/assets/00f02df6-5e6b-4aab-8188-02876b9d6c80)

**Comment by 1475Notary** [comment]
ok. Got your email. Give you a chance and support your this round.
![image](https://github.com/user-attachments/assets/253f5b66-c095-48af-9455-129ea2dc72fc)

**Comment by 1475Notary** [comment]
ok. Got your email. Give you a chance and support your this round.
![image](https://github.com/user-attachments/assets/44f596b8-0aed-4ecc-bd9c-8ebaa596aa3e)

**Comment by vance8sl** [comment]
@1475Notary Thank you!

**Comment by vance8sl** [comment]
@1475Notary 
Hello notary, we'd provided some new sps to our application.
f0114153 China
f01084941 HongKong
f03100002 China
f03100009 HongKong

**Comment by 1475Notary** [comment]
checker:manualTrigger

**Comment by 1475Notary** [comment]
checker:manualTrigger

**Comment by 1475Notary** [comment]
The average retrieval success rate looks low. How could that be?

**Comment by vance8sl** [comment]
@1475Notary Hello notary, it will be improved next time. When these sps continue to   seal, the rate will increase. Please look forward to new report.

**Comment by 1475Notary** [comment]
Ok, hopefully we'll see an improvement in the next tranche.

**Comment by 1475Notary** [comment]
checker:manualTrigger

**Comment by 1475Notary** [comment]
Good job, please continue to improve the  retrieval success rate.

**Comment by vance8sl** [comment]
checker:manualTrigger

**Comment by vance8sl** [comment]
@1475Notary Hello notary, after we saw the report you provided last week, we discussed the problem in it with sps. Then they fixed and upgraded their programs based on the problem.
Now I see that retrieval success rates are growing.
f03100008 Shenzhen
f01084413 Ningbo
f03214920 Hong Kong
f03161261 New York
f03192503 Hong Kong
f03189917 Dulles
f03156722 Hong Kong
I also looked for more sp's that fulfill the requirements to improve the retrieval success rate. Please waiting for the better results about our report.

**Comment by 1475Notary** [comment]
@vance8sl ok. Keep focusing on improving it.

**Comment by filecoin-watchdog** [comment]
checker:manualTrigger

**Comment by 1475Notary** [comment]
@vance8sl Hello, just come to ask for your storage situation these days. Are you continuing your storage plan?

**Comment by vance8sl** [comment]
@1475Notary Hello notary, we are updating our sp list and waiting for your approval for next round.
We've already stopped the coorperation with sps who still do not provide retrieval rate. 
![image](https://github.com/user-attachments/assets/2a0529dc-db9b-4a2e-82a1-030bd5bead22)
Please support us and we will give a better retrieval report.

**Comment by 1475Notary** [comment]
@vance8sl Good. In view of your previous progress, I'd like to continue to support your storage plan. I'm waiting for a new report.

**Comment by 1475Notary** [comment]
checker:manualTrigger

**Comment by 1475Notary** [comment]
By comparing the previous report with the current one, we found that the client has stopped working with the sps that did not support retrieval in the previous period. Other sps they are working with have good successful retrieval rates. 

@vance8sl Happy to see your progress. But you still need to pay attention to stop working with sp with low retrieval success rates in time. Support the next round.

**Comment by vance8sl** [comment]
@1475Notary Hi notary, we have invited more sps in our plan.
f03214920 HongKong
f03100008 Guangdong
f03156722 HongKong
f03189917 USA
f03161261 USA
f03192503 HongKong
f01084413 Singapore
f03224828 UK
f03242023 Shanghai
f01431043 HongKong
f03216485 Sichuan

**Comment by filecoin-watchdog** [comment]
checker:manualTrigger

**Comment by 1475Notary** [comment]
@vance8sl Are these sps worked with you to store data now?
>f03231666
f03241837
f01975299
f02826588
f02851143
f03066836

**Comment by vance8sl** [comment]
@1475Notary Yes. These days I'm too busy so I can't keep the list up to date. These sps are our new partners. But we've stopped with this sp f02826588 now. Other sps' retrieval rate is good. You can check them.

**Comment by 1475Notary** [comment]
@vance8sl 
 >30.08% of deals involve data replicated across fewer than 4 storage providers
 
 You also should explain it.

**Comment by vance8sl** [comment]
@1475Notary This is because some sps's sealing speed is slow. After some time, they will catch up with the other sps.

**Comment by 1475Notary** [comment]
@vance8sl The amount of replicas is more than you used to want to store in your application. It seems to be related to you adding new sps.

**Comment by vance8sl** [comment]
@1475Notary Yes. It takes time to communicate with more sp to reach cooperation. Sometimes sp and we cannot work together all the time for a variety of reasons, which may be theirs or ours. Sometimes some sp's don't support retrieval and we stop working with them to find other sps. In order to keep up the progress, we had to find other sp to continue to complete the storage when we met some sps.

**Comment by filecoin-watchdog** [comment]
@vance8sl 
>Yes. It takes time to communicate with more sp to reach cooperation. Sometimes sp and we cannot work together all the time for a variety of reasons, which may be theirs or ours. Sometimes some sp's don't support retrieval and we stop working with them to find other sps. In order to keep up the progress, we had to find other sp to continue to complete the storage when we met some sps.

Does this mean you will restart the entire process each time a Storage Provider (SP) fails to meet expectations, resulting in only partial data storage and creating countless replicas? Given that you declared 8 replicas but have already produced 20, how do you plan to achieve 8 full replicas with only 5 PiB available? 
Also, could you explain how you keep track of where each piece is stored, and how you determine which provider holds which data?

**Comment by vance8sl** [comment]
@filecoin-watchdog 
Our initial plan was to store 8 replicas with 5 PiB, but in actually working with the sp, we found that the sp would interrupt working with us due to a number of factors. Different sps have different storage speed, and then we look for new sps to continue storage in order to make the storage progress as consistent as possible across replicas. Due to the different situation of sps, we can't be sure now if we can use 5 PiB to store all 8 replicas anymore. The other main reason for frequent sp changes is that we want to follow the community rules of keeping data at the sps that can be retrieved.
When we prepare the data, we will record the cid of the data and the relevant information of the data content in the database. so that we can follow the storage and distribution of data.
